src/utils/ContainerUtils.ts

```diff
@@ -1,13 +1,14 @@
 export type SearchMethodOptions = "equals" | "contains";
 
 export interface CommonProps {
-    class?: string;
     defaultQuery: string;
     placeHolder: string;
     showSearchBar: boolean;
 }
 
 export interface OfflineSearchProps extends CommonProps {
+    class?: string;
+    mxform: mxui.lib.form._FormBase;
```

Comment:
I dont like the pattern of the `CommonProps` but is can left as is. We might need to discuss it before using it in the next widgets. It can left alone on the implementation.

Reply:
This is to avoid duplication of properties and making properties more reusable. Okay we shall discuss it in the next widget.